src/main/java/Duke.java

```diff
@@ -1,10 +1,44 @@
+import duke.helpers.Parser;
+import duke.helpers.Storage;
+import duke.helpers.TaskList;
+import duke.helpers.Ui;
+
+import java.io.PrintWriter;
+import java.util.Scanner;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileWriter;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+
+
+/**
+ * Duke Class: Serves as entry point of the program, or JAR file.
+ *
+ */
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        try {
+            Ui.doGreeting();
```

Comment:
The naming of the various variables and methods do follow the coding standards and I like how all the methods start with verbs as it expresses clearly what it is trying to do

Reply:
Thank you for the generous feedbacks! I agree, naming conventions really help to give a nicer touch to any piece of code. All the best in your future increments! 😄